### DISCUSSION TITLE
magick identify PNG files

### DISCUSSION THREAD

**Discussion by jetpack1116** [ImageMagick/ImageMagick#4951] [Help]
Using `magick identify` command, how to force it will display x y resolution in pixels-per-inches?

**Comment by fmw42** [comment]
For what file type?  PNG only shows resolution in pixels per centimeter.

**Reply by jetpack1116** [reply]
Thanks for the reply.

i just found that `identofy` can have `-units` to set what unit to get for the resolution.

`magick identify -units PixelsPerInch  -format 'resolutionX':'%x','resolutionY':'%y','Unit':'%U'  <PNG file path>`
